wski18n/resources/en_US.all.json

```diff
@@ -1558,5 +1558,21 @@
   {
     "id": "Cannot specify both --kind and --docker at the same time.",
     "translation": "Cannot specify both --kind and --docker at the same time."
+  },
+  {
+    "id": "Relative path '{{.path}}' does not include valid path parameters. Each parameter must be enclosed in curly braces{}.",
+    "translation": "Relative path '{{.path}}' does not include valid path parameters. Each parameter must be enclosed in curly braces{}."
```

Comment:
Still don't like `curly braces{}`. Maybe `curly braces '{}'`, or is that not as straight forward?

Reply:
actually, i like that '{}'.  I use that.